akka-remote/src/main/scala/akka/remote/Endpoint.scala

```diff
@@ -215,8 +215,6 @@ private[remote] class ReliableDeliverySupervisor(
   val autoResendTimer = context.system.scheduler.schedule(
     settings.SysResendTimeout, settings.SysResendTimeout, self, AttemptSysMsgRedelivery)
 
-  private var bufferWasInUse = false
```

Comment:
So this is basically just reverting that optimization (or well, half of it, TooLongIdle is untouched I guess)?